### PR TITLE
Add xterm to docker base

### PR DESCRIPTION
**Pull request by kkauder**
### Briefly, what does this PR introduce?
Add xterm to docker base

### What kind of change does this PR introduce?
- [ ] Bug fix (issue #__)
- [*] New feature (issue #13)
- [ ] Documentation update
- [ ] Other: __

### Please check if this PR fulfills the following:
- [ ] Tests for the changes have been added
- [ ] Documentation has been added / updated
- [ ] Changes have been communicated to collaborators

### Does this PR introduce breaking changes? What changes might users need to make to their code?
no

### Does this PR change default behavior?
no

### CODE REVIEW

File: containers/debian/base.Dockerfile [hunk 1]
```diff
@@ -63,6 +63,7 @@ apt-get -yqq install --no-install-recommends                            \
         unzip                                                           \
         vim-nox                                                         \
         wget
```
Comment (wdconinc) [inline containers/debian/base.Dockerfile:65]:
```suggestion
        wget \
```

**Review by wdconinc — CHANGES_REQUESTED** [review]

**Comment by wdconinc** [comment]
We try to keep the container small. Adding xterm isn't just xterm but the entire X library system, for a program that does nothing different from a terminal. I'm not sure we need this. Let me see if another X program can be added through slack where we already install the X libraries.

**Comment by kkauder** [comment]
> We try to keep the container small. Adding xterm isn't just xterm but the entire X library system, for a program that does nothing different from a terminal. I'm not sure we need this. Let me see if another X program can be added through spack where we already install the X libraries.

Seems like enough base libraries are there (otherwise root wouldn't produce graphics anyway). I get
```
apt install xterm
[...]
After this operation, 3,618 kB of additional disk space will be used.
Do you want to continue? [Y/n]
```

**Comment by wdconinc** [comment]
Adding it in the place where the x11 libraries are already installed, https://eicweb.phy.anl.gov/containers/eic_container/-/merge_requests/880.

At some point we'll get rid of more x11 libraries in base (we shouldn't need gv or poppler-utils in that layer; if you add it all up it's closer to 80MB in x11 libraries, which are then duplicated for the spack packages too).

**Pull request closed without merging**